scripts/pc2sandbox_interactive.sh

```diff
@@ -391,35 +447,46 @@ do
 		# Get wall time - we want it as close as possible to when we fetch the cpu time so they stay close
 		# since the cpu.stat includes the time this script takes AFTER the submission finishes.
 		endtime=`GetTimeInMicros`
```

Comment:
The comment above notwithstanding, the placement of the call to `GetTimeInMicros` here means that (especially with the addition of the call to `KillChildProcs`, above) the user is being charged for not only the time it takes to kill the child processes, but also the time it takes to do the REPORT calls, above.  It seems like the call to `GetTimeInMicros` should come as close as possible to the `wait` (inside the `while` loop, above).   (I'm not sure of the ramifications of moving it up, but I _am_ sure that with the call where it is the user is begin charged for system overhead...)

Reply:
Again, the user is not "charged" for the time.  We **REPORT** the time in the summary is all; no child process will be killed due to this value.